.github/workflows/maven-release.yml

```diff
@@ -15,62 +15,29 @@ on:
   workflow_dispatch:
     inputs:
       tag:
-        description: 'tag/version'
+        description: 'The existing tag (version) to be released (ex: 5.0.1).'
         required: true        
 
-      action_tag:
-        description: 'Create tag? ("no" to skip)'
+      create-github-release:
+        description: 'Create GitHub release with tag? ("false" to skip)'
+        type: boolean
         required: true
-        default: 'yes'
+        default: true
+
+      edge-identity-dependency:
+        description: 'The AEPEdgeIdentity dependency version in gradle.properties to be validated (ex: 3.0.1).'
```

Comment:
nit: Here we use ex: 3.0.1 the [L18 tag](https://github.com/adobe/aepsdk-edge-android/pull/171/files#diff-1e1110e4b87ab2617e8052089ef9e5b403618114a3cf009ead9b3925d3bc89aaR18) has ex: 5.0.1 
 
 let's keep example versions also consistent and probably to latest major version i.e. 3.x

Reply:
works